langchain/src/memory/vector_store.ts

```diff
@@ -14,6 +14,8 @@ export interface VectorStoreRetrieverMemoryParams {
   outputKey?: string;
   memoryKey?: string;
   returnDocs?: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  metadata?: Record<string, any>;
```

Comment:
_nit_: consider using `unknown` instead to avoid unintended disabling of type-checking

Reply:
good call, i'll update this part